.github/scripts/ep_skill_config.py

```
@@ -0,0 +1,31 @@
+"""Build agentic-ci SkillConfig for EP review."""
+
+try:
+    from agentic_ci.skill import SkillConfig
+except ImportError:
+    SkillConfig = None
+
+
+def build_skill_config(hooks, skill_name, skills_path, skill_path):
```

Ruff (0.15.20)

[warning] Unused function argument: `skill_path`

(ARG001)